plugin/trino-hive/src/main/java/io/trino/plugin/hive/fs/TrinoFileSystemCacheStats.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.fs;
+
+import io.airlift.stats.CounterStat;
+import org.weakref.jmx.Managed;
+import org.weakref.jmx.Nested;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public class TrinoFileSystemCacheStats
+{
+    private final CounterStat getCalls = new CounterStat();
```

Comment:
Do we need to capture the timings as well? (`TimeStat`)

cc @sopel39

Reply:
Can it be a long call? If it's atomic more or less, then I don't see a point

Reply:
I don't know the answer.

Reply:
We don't have a use case for tracking the timings in any of our deployments so unless you feel strongly, I won't capture them in this PR.